PythonPonies/bangazon_cli
# removed bangazon.db + updated SQL file + update gitignore

# Description
This pull request includes the following changes:

- Removed 'last_name' from Customer table in populatedb.sql
- Changed 'first_name' to 'customer_name' on Customer table in populatedb.sql
- Removed bangazon.db file
- Updated .gitignore to include '*.db'

## Number of Fixes
3 fixes. 

## Related Ticket(s)
#1 + #19 

## Problem to Solve
This pull request solves the following problems:

- Updates gitignore to not see '.db' files
- Changes Customer table first and last name to one simple customer name
- Removes bangazon.db from repository so that each team member can work with the database on their local machines. 

## Expected Behavior
There should no longer be a database file in the repository.

There should only be one field for customer name in Customer table. 

Git should no longer see '.db' files. 

## Steps to Test Solution

1. git fetch
2. git checkout swh-sql-fix
3. open populatedb.sql + .gitignore
4. ensure that proposed changes are complete
5. ensure that bangazon.db is no longer present in branch. 

## Testing

[ ] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[ ] I certify that all existing tests pass

## Documentation
[ ] There is new documentation in this pull request that must be reviewed..
[ ] I added documentation for any new classes/methods

## Deploy Notes
N/A
